### DISCUSSION TITLE
应该如何正确设置 ALLOWED_DOMAIN？

### DISCUSSION THREAD

**Discussion by onmpen** [imsyy/DailyHotApi#20] [Q&A] [answered]
### 问题描述

我的 api 地址：api.daydayday.com
前端地址：daydayday.com
.env 里的 只有设置 ALLOWED_DOMAIN = '*' 时才能正确获取到信息

而设置为  ALLOWED_DOMAIN = 'api.daydayday.com'  时，会提示：{"code":403,"message":"请通过正确的域名访问"}

**Comment by imsyy** [comment, marked as answer]
ALLOWED_DOMAIN 是用于识别前端页面域名的，填写后只允许该域名调用，并且要正确填写协议头
比如你的前端地址是  hot.daydayday.com，那么 ALLOWED_DOMAIN = "https://hot.daydayday.com"
